config/sample-input-tf-enterprise.tfvars

```diff
@@ -0,0 +1,99 @@
+# !NOTE! - These are only a subset of CONFIG-VARS.md provided for sample.
+# Customize this file to add any variables from 'CONFIG-VARS.md' that you want 
+# to change their default values.
+
+# ****************  REQUIRED VARIABLES  ****************
+# These required variables' values MUST be provided by the User
+#
+# NOTE: For Terraform Cloud/Enterprise these variables should be defined
+#       in the workspace in terraform cloud/enterprise as a terraform variable
+#
+#       You also need to define: GOOGLE_CREDENTIALS as an envrionment
+#       variable.
+# prefix         = "<prefix-value>"
+# location       = "<gcp-zone-or-region>" # e.g., "us-east1-b"
+# project        = "<gcp-project>"
+# ssh_public_key = "<ssh-public-key>"
+# ****************  REQUIRED VARIABLES  ****************
+
+#
+# !NOTE! - Without specifying your CIDR block access rules, ingress traffic
+#          to your cluster will be blocked by default.
+#
+# If you are not working from a machine on the SAS Cary Network, you can always use your own public ip:
+# Use the IP reported in https://ifconfig.me/ and append "/32", e.g. 1.2.3.4/32
+# For a list of the CIDRs of other SAS networks, see http://mom.unx.sas.com/net/InetAddrs.html
```

Comment:
This an be removed.

Reply:
What can be removed? TF Enterprise requires that the default_public_access_cidrs value be 0.0.0.0/0 given we don't know the IP of the worker agent that will actually being performing the updates inside the cluster. Please clarify

Reply:
The comment block mentions the "SAS Cary Network" and an internal URL.

You could probably leave the "Use the IP reported in https://ifconfig.me/ and append "/32", e.g. 1.2.3.4/32" portion though

Reply:
I can remove that
done